ui/src/album/AlbumListActions.js

```diff
@@ -37,26 +92,9 @@ const AlbumListActions = ({
           filterValues,
           context: 'button',
         })}
-      <ButtonGroup
-        variant="text"
-        color="primary"
-        aria-label="text primary button group"
-      >
-        <Button
-          size="small"
-          color={albumView.grid ? 'primary' : 'secondary'}
-          onClick={() => dispatch(albumViewGrid())}
-        >
-          <ViewModuleIcon fontSize="inherit" />
-        </Button>
-        <Button
-          size="small"
-          color={albumView.grid ? 'secondary' : 'primary'}
-          onClick={() => dispatch(albumViewList())}
-        >
-          <ViewHeadlineIcon fontSize="inherit" />
-        </Button>
-      </ButtonGroup>
+      {isSmall && (
```

Comment:
In the case of Small screens, we need to put back the original grid/list selector, as the ToggleFieldsMenu is hidden